web/src/message_view_header.ts

```diff
@@ -170,6 +170,54 @@ export function initialize(): void {
 export function render_title_area(): void {
     const filter = narrow_state.filter();
     build_message_view_header(filter);
+
+    $(".narrow_description").on("mouseenter mouseleave", function (event) {
+        const $description = $(this);
+        const windowWidth = $(window).width()!;
+        let hoverTimeout;
+
+        if (event.type === "mouseenter") {
+            if (!$description.hasClass("narrow_description_extended")) {
+                const currentWidth = $description.outerWidth();
+                // Set fixed width for word-wrap to work
+                $description.css("width", currentWidth + "px");
```

Comment:
Why is this width only computed if it's not already displayed? What happens if the window is resized while one of these descriptions are showing?

Reply:
I guess it's pretty hard to resize without losing the hover on it and thus hiding the component.